lib/style/module_directives.ex

```diff
@@ -213,7 +234,77 @@ defmodule Styler.Style.ModuleDirectives do
         |> Zipper.update(&Zipper.replace_children(&1, directives))
         |> Zipper.down()
         |> Zipper.rightmost()
-        |> Zipper.insert_siblings(nondirectives)
+        |> Zipper.insert_siblings(use_aliases(nondirectives, to_alias))
+    end
+  end
+
+  defp find_extractable_aliases(ast, aliases) do
+    aliased =
+      aliases
+      |> Enum.flat_map(fn
+        {:alias, _, [{:__aliases__, _, aliases}]} -> [aliases]
+        _ -> []
+      end)
+      |> MapSet.new(&List.last/1)
+
+    excluded_first = MapSet.union(aliased, @excluded_namespaces)
+    excluded_last = MapSet.union(aliased, @stdlib)
+
+    ast
+    |> Zipper.zip()
+    |> Zipper.traverse_while(%{}, fn
+      # Credo doesn't look at :@ nodes. I kinda like it? especially for typespecs. otoh, we should just Zipper.remove those 🤣
+      {{directive, _, _}, _} = zipper, acc when directive in ~w(use)a ->
+        {:skip, zipper, acc}
```

Comment:
https://github.com/rrrene/credo/issues/133 if this was ever the case, it certainly isn't now. so i think this function head can just go away